### PR TITLE
Add emodel-generalisation

### PR DESCRIPTION
**Pull request by ilkilic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ A suite to handle single-cell electrophysiological data and to build and validat
 - [Currentscape](#currentscape) - Plot currents in electrical models
 - [SSCxEModelExamples](#SSCxEModelExamples) - Reproduce Reva, Maria, et al. "A universal workflow for creation, validation and generalization of detailed neuronal models" (2023).
 - [e-model-packager](#e-model-packager) - Creates e-model packages from circuits
+- [emodel-generalisation](#emodel-generalisation) - Generalisation of electrical models of neurons with MCMC
```
Comment (arnaudon) [inline README.md]:
```suggestion
- [emodel-generalisation](#emodel-generalisation) - Generalisation and MCMC sampling of electrical models of neurons
```

File: README.md [hunk 2]
```diff
@@ -175,3 +176,11 @@ The models created by e-model-packager can be run easily using [EModelRunner](ht
 Useful links:
 [GitHub repo](https://github.com/BlueBrain/e-model-packager),
 [Documentation](https://e-model-packager.readthedocs.io/en/latest/).
+
+## emodel-generalisation
+
+Generalisation of neuronal electrical models on a morphological population with Markov Chain Monte-Carlo.
```
Comment (arnaudon) [inline README.md]:
```suggestion
Generalisation and MCMC sampling of neuronal electrical models.
```

File: README.md [hunk 3]
```diff
@@ -175,3 +176,11 @@ The models created by e-model-packager can be run easily using [EModelRunner](ht
...
+
+Useful links:
+[GitHub repo](https://github.com/BlueBrain/emodel-generalisation),
+[Documentation](https://emodel-generalisation.readthedocs.io/en/latest/).
```
Comment (darshanmandge) [inline README.md:188]: I think you can also add the link and [citation](https://github.com/BlueBrain/emodel-generalisation?tab=readme-ov-file#emodel-generalisation) of the paper here.

**Comment by ilkilic** [comment]
Thank you all, I have added your suggestions

**Review by darshanmandge — APPROVED** [review]

**Pull request merged**